tests/nextbutton.spec.ts

```diff
@@ -1,115 +1,91 @@
-//     await page.click('button[class="next-button"]');
-//     await page.waitForNavigation();
-//     await expect(page.locator('button[class="next-button"]')).toBeVisible();
-//   });
-
-//   it("should go to the Card Flip Game page when clicked on Joke Randomiser page", async () => {
-//     await page.click('button[class="next-button"]');
-//     await page.waitForNavigation();
-//     await page.click('button[class="next-button"]');
-//     await page.waitForNavigation();
-//     await expect(page.locator('button[class="next-button"]')).toBeVisible();
-//   });
-
-//   it("should go to the Rate Joke page when clicked on Card Flip Game page", async () => {
-//     await page.click('button[class="next-button"]');
-//     await page.waitForNavigation();
-//     await page.click('button[class="next-button"]');
-//     await page.waitForNavigation();
-//     await page.click('button[class="next-button"]');
-//     await page.waitForNavigation();
-//     await expect(page.locator('button[class="next-button"]')).toBeVisible();
-//   });
-
-//   it("should go back to the home page when clicked on Rate Joke page", async () => {
-//     await page.click('button[class="next-button"]');
-//     await page.waitForNavigation();
-//     await page.click('button[class="next-button"]');
-//     await page.waitForNavigation();
-//     await page.click('button[class="next-button"]');
-//     await page.waitForNavigation();
-//     await page.click('button[class="next-button"]');
-//     await page.waitForNavigation();
-//     await expect(page.locator('button[class="next-button"]')).toBeVisible();
-//   });
-// });
-
-// import { chromium, test, expect } from "@playwright/test";
-// import describe from "@playwright/test";
-
-// describe("BackButtonWrapper", () => {
-//   test("clicking the button navigates to the specified route", async () => {
-//     // Launch a new Chromium browser instance
-//     const browser = await chromium.launch();
-
-//     // Create a new page in the browser
-//     const page = await browser.newPage();
-
-//     // Navigate to the page with the BackButtonWrapper component
-//     await page.goto("http://localhost:5173/back-button-wrapper");
-
-//     // Click the button
-//     await page.click("button");
-
-//     // Get the URL of the current page
-//     const url = await page.url();
-
-//     // Assert that the URL is equal to the specified route
-//     expect(url).toBe("http://localhost:5173/next-page");
-
-//     // Close the browser
-//     await browser.close();
-//   });
-// });
+import { chromium, expect, test } from "@playwright/test";
+
+test.describe("Button component is visible on multiple pages", () => {
+  let browser;
+  let page;
+
+  test.beforeAll(async () => {
+    browser = await chromium.launch({ headless: false, slowMo: 3000 });
+  });
+
+  test.afterAll(async () => {
+    await browser.close();
+  });
+
+  test.beforeEach(async () => {
```

Comment:
Good refactor for visual UI confirmation that the beforeEach function opens a browser on a new page whenever the next button is clicked 👍🏼 👀